src/components/Download/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { type CommonProps } from 'types/components';
+import EContainer from '../EContainer';
+import { Button, Typography, Divider, Stack } from '@mui/material';
+import DownloadIcon from '@mui/icons-material/Download';
+
+interface DownloadItem {
+  label: string;
+  fileName?: string;
+  href: string;
+}
+
+export interface IDownolad extends CommonProps {
+  title: string;
+  subtitle?: string;
+  type?: 'button' | 'link';
+  items: DownloadItem[];
+}
+
+const Download = (props: IDownolad) => {
+  const { title, subtitle, items, theme, type } = props;
+  const backgroundColor = theme === 'dark' ? 'primary.dark' : 'white';
+  const textColor = theme === 'dark' ? 'primary.contrastText' : 'text.primary';
+  const isTypeButton = type === 'button';
+  return (
+    <EContainer direction="column" background={backgroundColor} py={10}>
+      <Typography variant="h2" mb={1} color={textColor}>
+        {title}
+      </Typography>
+      {subtitle && (
+        <Typography variant="body1" mb={1} color={textColor}>
+          {subtitle}
+        </Typography>
+      )}
+      <Divider />
+      <Stack alignItems={'flex-start'}>
+        {items.map((item, index) => (
+          <Button
+            key={`${item.label}-${index}`}
+            href={item.href}
+            variant={isTypeButton ? 'outlined' : 'text'}
+            startIcon={<DownloadIcon />}
+            color={theme === 'dark' ? 'negative' : 'primary'}
+            download={item.fileName ?? item.label}
```

Comment:
Maybe it's better to leave empty the value here in `download` attribute, when `fileName` is empty,  to keep the original filename of remote resource.
Rif https://www.w3schools.com/tags/att_a_download.asp